src/js/view/components/nextuptooltip.js

```diff
@@ -202,22 +203,13 @@ define([
 
             this.offset = offset;
         },
+        onRelatedCached: function() {
+          this.enableRelatedMode();
```

Comment:
Neither cached nor playlist will fire when there are no related items, but I can add a check. However, the bug related to the Next button flashing is caused by playlists with a length of 1. Going to rework that in a future ticket.